templates/partials/question.html

```diff
@@ -14,6 +14,9 @@
 {% set question_instruction = format_paragraphs(question.instruction) %}
 {% set question_error = form.question_errors[question.id] %}
 
+{%- if content.content -%}
+  <h1>{{ content.content.title }}</h1>
+{%- endif -%}
```

Comment:
How come this was added?

Reply:
We were experimenting with templates and forgot to remove it after, fixed now.